src/reducers/index.js

```diff
@@ -0,0 +1,19 @@
+import { combineReducers } from 'redux';
+import books from './books';
+
+const initialState = {
+
+  books: [
+    { id: Math.floor(Math.random() * 5), title: 'Title one', category: 'Sci-Fi' },
+    { id: Math.floor(Math.random() * 5), title: 'Title Two', category: 'Action' },
+    { id: Math.floor(Math.random() * 5), title: 'Title Three', category: 'Kids' },
+  ],
+};
+
```

Comment:
I guess you missed this, the state should be initialized inside `src/index.js` not inside `reducers`
![image](https://user-images.githubusercontent.com/31897434/79816439-2862c880-837b-11ea-8be7-50ec7360aa1c.png)